src/globals.cpp

```diff
@@ -63,12 +63,14 @@ int BetterSMS::getScreenOrthoWidth() {
     case AspectRatioSetting::WIDE:
         return 853;
     case AspectRatioSetting::ULTRAWIDE:
-        return 1120;
+        return 1137;
     }
 }
 
 f32 BetterSMS::getScreenToFullScreenRatio() { return static_cast<f32>(getScreenRenderWidth()) / 600.0f; }
 f32 BetterSMS::getScreenRatioAdjustX() { return (getScreenToFullScreenRatio() - 1.0f) * 600.0f; }
+f32 BetterSMS::getScreenScale() { return gAspectRatioSetting.getInt() == AspectRatioSetting::FULLOPENMATTE ? 0.75f : 1.0f; }
+f32 BetterSMS::getCalculatedFovy(f32 fov, f32 zoom) { return halfRadiansToAngle(atanf(tanf(angleToHalfRadians(fov)) * zoom)); }
```

Comment:
`angleToHalfRadians` should be `halfAngleToRadians` afaik. Also, include "libs/constmath.hxx" to have these functions available.